tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -709,6 +711,21 @@ def fit(
             setting the memory limit (and any other resources) on systems with shared resources that are controlled by the operating system (e.g., SLURM and
             cgroups). Otherwise, AutoGluon might wrongly assume more resources are available for fitting a model than the operating system allows,
             which can result in model training failing or being very inefficient.
+        callbacks : List[AbstractCallback], default = None
+            [Experimental] Callback support is preliminary, targeted towards developers, and is subject to change.
```

Comment:
Nit: Should we use [admonitions](https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html) to highlight experimental features?

```
:::{warning}
This is an experimental feature and may change in the future releases without warning.
:::
```

Reply:
Nice! Added